migration20170323_security-question.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE `pool_ref`.`security_question` (
+  `id` INT(11) NOT NULL AUTO_INCREMENT,
+  `question` VARCHAR(80) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `id_UNIQUE` (`id` ASC));
+
+ALTER TABLE `pool_ref`.`user`
+ADD COLUMN `security_question_id` INT(11) NULL AFTER `hashedpassword`,
+ADD COLUMN `security_question_answer` VARCHAR(80) NULL AFTER `security_question_id`;
+
+ALTER TABLE `pool_ref`.`user`
+CHANGE COLUMN `hashedpassword` `hashed_password` VARCHAR(80) NOT NULL ;
+
+INSERT INTO `pool_ref`.`security_question` (id, question)
```

Comment:
INSERT methods can perform multiple lines at once

This can be simplified and also would perform substantially less requests on migration